Doc/library/statistics.rst

```diff
@@ -643,23 +643,23 @@ However, for reading convenience, most of the examples show sorted sequences.
 
    .. versionadded:: 3.10
 
-.. function:: linear_regression(regressor, dependent_variable)
+.. function:: linear_regression(independent_variable, dependent_variable)
 
    Return the intercept and slope of `simple linear regression
    <https://en.wikipedia.org/wiki/Simple_linear_regression>`_
    parameters estimated using ordinary least squares. Simple linear
-   regression describes the relationship between *regressor* and
+   regression describes the relationship between *independent_variable* and
```

Comment:
Add articles to the sentence:  between AN independent variable and A dependent variable.

Also the example on line 675 needs to be updated to reflect the new slope/intercept ordering.

Reply:
I've updated the example on line 675 (now line 676).

I made some additional changes to 651:
... regression describes the relationship between an independent variable *x* and a dependent variable *y* ...